ui/components/ui/nft-info-setApprovalForAll/nft-info-setApprovalForAll.js

```diff
@@ -0,0 +1,157 @@
+import React, { useContext } from 'react';
+import PropTypes from 'prop-types';
+import { useSelector } from 'react-redux';
+import { I18nContext } from '../../../contexts/i18n';
+import Box from '../box';
+import Typography from '../typography';
+import {
+  COLORS,
+  DISPLAY,
+  FONT_WEIGHT,
+  TYPOGRAPHY,
+  TEXT_ALIGN,
+} from '../../../helpers/constants/design-system';
+import Identicon from '../identicon';
+import Tooltip from '../tooltip/tooltip';
+import { getTokenList } from '../../../selectors';
+
+export default function NftInfoSetApprovalForAll({
+  assetName,
+  tokenAddress,
+  total,
+  collections = {},
+}) {
+  const t = useContext(I18nContext);
+  const tokenList = useSelector(getTokenList);
+
+  const nftTokenListImage = tokenList[tokenAddress.toLowerCase()]?.iconUrl;
+
+  let nftCollectionNameExist;
+  let nftCollectionImageExist;
+
+  Object.values(collections).forEach((nftCollections) => {
+    if (nftCollections.collectionName === assetName) {
+      nftCollectionNameExist = nftCollections.collectionName;
+      nftCollectionImageExist = nftCollections.collectionImage;
+    }
+  });
+
+  const renderCollectionImage = (collectionImage, collectionName, key) => {
```

Comment:
Perhaps rename this to `renderCollectionImageOrName`

Reply:
It looks like the `key` param is never used

Reply:
Hi @danjm  , You can see changes [here](https://github.com/MetaMask/metamask-extension/pull/15844/files#diff-ec98920531d74df41855f0a46de3c187b84296d0ca2a6cc2e67548d48db837e9)